members/members/ClJarvis.ts

```diff
@@ -47,4 +47,10 @@ export const ClJarvis: MemberObject = {
 		},
 	],
 	badges: ['Hacktoberfest2022'],
+	['Hacktoberfest2023'],
```

Comment:
Let's make this one array with two strings 👍

```suggestion
	badges: ['Hacktoberfest2022', 'Hacktoberfest2023'],
```